package.json

```diff
@@ -16,12 +19,19 @@
     "request": "^2.73.0"
   },
   "devDependencies": {
+    "babel-eslint": "^6.1.2",
     "coffee-script": "^1.10.0",
-    "jshint": "^2.9.2"
+    "eslint": "^3.1.1",
+    "eslint-config-airbnb-base": "^4.0.2",
+    "eslint-plugin-import": "^1.11.1",
+    "eslint-plugin-promise": "^2.0.0",
+    "nodemon": "^1.9.2"
   },
   "scripts": {
     "test": "npm run lint",
-    "lint": "jshint --exclude-path .gitignore ."
+    "lint": "eslint ./lib",
```

Comment:
This wouldn't cover the root folder, which currently consists of a minimal `index.js`. If we stick to use that only as an entry point, I guess there's no linting to go wrong...